rand_chacha/Cargo.toml

```diff
@@ -20,11 +20,12 @@ appveyor = { repository = "rust-random/rand" }
 
 [dependencies]
 rand_core = { path = "../rand_core", version = "0.4" }
-c2-chacha = { version = "0.2", default-features = false }
+c2-chacha = { version = "0.2.2", default-features = false }
 
 [build-dependencies]
 autocfg = "0.1"
 
 [features]
-default = ["std"]
+default = ["std", "simd"]
```

Comment:
This is not a breaking change, because the `simd` feature does not change the API, right?

Reply:
Right, disabling `simd` just forces use of the same portable implementation as architectures that don't have SIMD backends.